rel8-jdk8-overrides.yaml

```diff
@@ -0,0 +1,41 @@
+schema_version: 1
```

Comment:
Note the name of this file should be something like rel-jdk8-rhel8-overrides.yaml. The rel- at the start really is short for 'Release', meaning it is an override used when building release images.

Reply:
Ok I will rename the files

Reply:
Done in [commit](https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/497/commits/322883892178e4303c652cb0448d35781f9232f1)